### PR TITLE
Make Chrome 28+ notifications working

### PR DESCRIPTION
**Pull request by deathbeam**
According to this [stackoverflow post](http://stackoverflow.com/questions/23820931/after-updating-chrome35-0-1916-114-m-webkitnotifications-does-not-work) and this [discussion](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/8vqyfHa8_dw) window.webkitNotifications was removed in Chrome and was replaced with chrome.notifications, so this patch should make notifications in chrome work again.

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "content_scripts": [
         {
             "matches": [
+                "*://lastfm-img2.akamaized.net/*",
```
Comment (cgravolet) [inline src/manifest.json]: Why was this added?

File: src/js/main-background.js [hunk 2]
```diff
@@ -319,6 +320,25 @@ window.scroblrGlobal = (function () {
                 }, 5000);
             }
         }
+        
+        // Chrome 28+ notifications
+        if (chrome && getOptionStatus("notifications")) {
```
Comment (cgravolet) [inline src/js/main-background.js]: Did chrome completely deprecate the `webkitNotifications` API? Just wondering if it's possible for the above condition and this one to both be true, resulting in 2 notifications being sent?

Comment (deathbeam) [inline src/js/main-background.js]: Oh, good point. For some time webkitNotifications and chrome.notifications was both present in Chrome, so yes, if someone have Chrome version which supports both, then yes, 2 notifications will be sent. Going to adjust the patch then.

File: src/manifest.json (hunk 1 above)
Comment (deathbeam) [inline src/manifest.json]: Because when getting the album art from Spotify, it is getting it from lastfm-img2 server, and without this, chrome.notifications API will fail to load album art and throw exception.
